src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheck.java

```diff
@@ -101,18 +101,75 @@
  * &lt;module name=&quot;ParenPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Foo {
+ *
+ *   public void foo() {  // OK
+ *     bar( 1);  // violation, space after left parenthesis
+ *   }
+ *
+ *   public void bar(int k ) {  // violation, space before right parenthesis
+ *     while (k > 0) {  // OK
+ *     }
+ *
+ *     Test obj = new Test(k);  // OK
+ *   }
+ *
+ *   public void fun() {  // OK
+ *     switch( y ) {  // violation, spaces before and after parenthesis
```

Comment:
`spaces before and after parenthesis`
please specify what parens you mean

Reply:
According to the default configuration, there should not be any space:
a) after a left parenthesis,
b) before a left parenthesis.

So, in this case, there is a space after ```switch( ``` and before ```y )```, which causes 1st violation according to (a) and,
there is a space after ```switch( y``` and before ```)```, which causes the 2nd violation in the same line according to (b).

Reply:
Should I split this violation into separate violation for left and right parens?